pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -174,6 +174,8 @@ public ServerType getServerType() {
     public static final String CONFIG_OF_STARTER_ENABLE_SEGMENTS_LOADING_CHECK =
         "pinot.server.starter.enableSegmentsLoadingCheck";
     public static final String CONFIG_OF_STARTER_TIMEOUT_IN_SECONDS = "pinot.server.starter.timeoutInSeconds";
+    public static final String CONFIG_OF_STARTER_REALTIME_CONSUMPTION_CATCHUP_WAIT_TIME_SECONDS =
```

Comment:
I would put this along with the **CONFIG_OF_SERVER_MIN_RESOURCE_PERCENT_FOR_START** since both of them are for the status check instead of the starter. Maybe rename it to "pinot.server.startup.realtimeConsumptionCatchupTimeMs"?